### ISSUE TITLE
No sign for required curl version.

### ISSUE THREAD

**Issue by GitSparTV** [libcpr/cpr#1119]
### Description

On ubuntu 22.04 the latest curl is 7.81.0. I get no warnings but a compilation error:

```cpp
/cpr/cpr/error.cpp:131:14: error: use of undeclared identifier 'CURLE_UNRECOVERABLE_POLL'
  131 |         case CURLE_UNRECOVERABLE_POLL:
      |              ^
/cpr/cpr/error.cpp:133:14: error: use of undeclared identifier 'CURLE_TOO_LARGE'
  133 |         case CURLE_TOO_LARGE:
      |              ^
2 errors generated.
```

### Example/How to Reproduce

install curl from apt on ubuntu and try `cmake .. -DCPR_USE_SYSTEM_CURL=ON -DBUILD_SHARED_LIBS=OFF && cmake --build .`

### Possible Fix

Tell the supported version in README or make a check in CMake or make macro check for curl version in the code

### Where did you get it from?

GitHub (branch e.g. master)

### Additional Context/Your Environment

- OS: Ubuntu
- Version: 22.04 (Docker)

**Comment by COM8** [comment]
Thanks for reporting!
I need to go through all errors and add version checks.

**Comment by COM8** [comment]
Fixed in #1120 and will be included in Release 1.11.0 and 1.9.8
